Swiftfin tvOS/Components/PosterButton.swift

```diff
@@ -149,6 +149,7 @@ extension PosterButton {
                         .font(.footnote)
                         .fontWeight(.regular)
                         .foregroundColor(.primary)
+                        .padding(.top, 1)
```

Comment:
I know that it's annoying that the text is slightly covered when focused, but this doesn't really help either and we shouldn't add this. What I ideally want is something like [TVPosterView](https://developer.apple.com/documentation/tvuikit/tvposterview) from `TVUIKit`. These views can't be wrapped and I had attempted to make a recreation however I ran into horrendous performance issues with `FocusState`. However, I wonder if my work in #905 would make a recreation viable again.

Apple has left tvOS + SwiftUI in the dust - it's very frustrating.

Reply:
So why exactly does adding padding not help? From my testing it does allow for the focused item to not cover the text without any major side effect besides the text being a little lower that before. Is that the issue? Maybe it's not a final solution but it fixes the issue and is easily removed when a better solution is found. Thanks.

Reply:
The issue is the total design, it makes additional padding that doesn't look nice. Sure it fixes the button from overlapping the text as I said the ideal solution is a design that accommodates both focused and non-focused states.

Reply:
I think it looks nice but agree that a design that accommodates both focused and unfocused would be best. I have removed that code.